src/lib.rs

```diff
@@ -139,6 +139,45 @@ pub(crate) const CRINGE_ESLINT_DISABLE: &str = "/* eslint-disable */
 // #[cfg(doctest)]
 // doc_comment::doctest!("../README.md");
 
+/// The configuration for the generator
+#[derive(Default)]
+pub struct ExportConfiguration {
```

Comment:
Can you also move this into `src/ts.rs`, as it holds a `specta::ts::*` type that is really only applicable to Typescript exporting. In Tauri Specta JS exporting is a bit special and leaches off the TS stuff so it may seem confusing but this is design to accommodate support for future languages that don't leach off the TS exporter.